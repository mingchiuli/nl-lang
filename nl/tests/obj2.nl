p=122;

a={
   a:12,b:13,c:{
      o:p,key:"haha"
   }
};

println(a);

a.c.key="hehe";
println(a);