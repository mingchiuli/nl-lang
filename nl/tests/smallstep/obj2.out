{a:12,b:13,c:{key:haha,o:122}}
{a:12,b:13,c:{key:hehe,o:122}}
